pkg/porter/invoke_test.go

```diff
@@ -3,10 +3,48 @@ package porter
 import (
 	"testing"
 
+	"github.com/cnabio/cnab-go/bundle"
+	"github.com/cnabio/cnab-go/bundle/definition"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func buildExampleBundle() bundle.Bundle {
```

Comment:
nit: this function appears to not be used here, but rather in the lifecycle_integration_test... should it move there?